Dockerfile

```diff
@@ -1,31 +1,10 @@
 FROM jwilder/nginx-proxy
-MAINTAINER [ "David Parrish <daveparrish@tutanota.com", "Yves Blusseau <90z7oey02@sneakemail.com>", "Hadrien Mary <hadrien.mary@gmail.com>" ]
 
-RUN apt-get update
-
-# Install python packages needed by simp_le
-RUN apt-get install -y -q --no-install-recommends python python-requests
-
-# Install python packages needed to build  simp_le
-RUN apt-get install -y -q --no-install-recommends git gcc libssl-dev libffi-dev python-dev python-pip
-
-# Get Let's Encrypt simp_le client source
-RUN git -C /opt clone https://github.com/kuba/simp_le.git
-
-WORKDIR /opt/simp_le
-# Upgrade setuptools
-RUN pip install -U setuptools
-# Install simp_le in /usr/local/bin
-RUN python ./setup.py install
-
-# Make house cleaning
-RUN rm -rf /opt/simp_le
-
-RUN apt-get autoremove -y git gcc libssl-dev libffi-dev python-dev python-pip
-
-RUN apt-get clean all
-RUN rm -r /var/lib/apt/lists/*
+MAINTAINER Jason Wilder <mail@jasonwilder.com>
```

Comment:
I think this maintainer should be me "David Parrish <daveparrish@tutanota.com>". Jason is not maintaining this Dockerfile.